### DISCUSSION TITLE
Play 2.8.18

### DISCUSSION THREAD

**Discussion by mkurz** [playframework/playframework#11493] [Announcements]
The Play Team is happy to announce the release of Play 2.8.18.

## :green_book: What is new?

This release includes a fix for [#10590](https://github.com/playframework/playframework/issues/10590) to avoid StackOverflowErrors when sending `multipart/form-data` requests with many fields as payload. We recommend upgrading as soon as possible.

Following pull requests got merged for this release:

- #11491 [2.8.x] Upgrade Akka, Netty, Spring, Joda-Time by @mkurz
- #11360 To refactor `BodyPartParser` of `Multipart` in order to avoid StackOverflowError easily by @yousuketto
- #11465 [2.8.x] Update MIME types (backport #11406) by @PromanSEW
- #11464 [2.8.x] Upgrade netty-reactive-streams by @mkurz
- #11459 [2.8.x] Latest scala sbt versions by @mkurz

For more details see the [full list of changes](https://github.com/playframework/playframework/compare/2.8.17...2.8.18/) and the [2.8.18 milestone](https://github.com/playframework/playframework/issues?page=1&q=milestone%3A%222.8.18%22).

## :heart: Thanks to our premium sponsors!

<div align="center">
  <a href="https://informaticon.com/"><img src="https://www.playframework.com/assets/images/home/sponsors/49a3d5258c8e8b1daf5481eb9e00b898-informaticon-logo-black.png" width="250"></a>
  <a href="https://cedarlakeventures.com/"><img src="https://www.playframework.com/assets/images/home/sponsors/bec2b526c9ce52c051f9089a10044867-cedar-lake-ventures.png" width="250"></a>
  <a href="https://iterable.com/"><img src="https://www.playframework.com/assets/images/home/sponsors/61ddb4c3665b621e6672181f97196748-iterable.png" width="250"></a>
  <a href="https://pronto.net/"><img src="https://www.playframework.com/assets/images/home/sponsors/c77b1d664f10a1c9cb19b97c6d8bd204-pronto-software.png" width="250"></a>
  <a href="https://civiform.us/"><img src="https://www.playframework.com/assets/images/home/sponsors/cb047b3782866c962c4d6a35b056b809-civiform.png" width="250"></a>
  <a href="https://theguardian.com/"><img src="https://www.playframework.com/assets/images/home/sponsors/b15eb0f249dbc45089872e268d8ea5ad-the_guardian.png" width="250"></a>
</div>

If you find this OSS project useful for work, please consider asking your company to support it by <a href="https://www.playframework.com/sponsors">becoming a sponsor</a>.
You can also individually sponsor the project by <a href="https://www.playframework.com/sponsors">becoming a backer</a>.

<div align="center">
  <a href="https://opencollective.com/playframework" target="_blank">
    <img src="https://opencollective.com/playframework/donate/button@2x.png?color=blue" width="250" />
  </a>
</div>

## :bow: Thanks to our contributors

Finally, thanks to the community for their help with detailed bug reports, discussions about new features and pull request reviews. This project is only possible due to the help we had from amazing contributors.
Special thanks to all code contributors who helped with this particular release (they are listed below)!

<hr /><em>This discussion was created from the release <a href='https://github.com/playframework/playframework/releases/tag/2.8.18'>Play 2.8.18</a>.</em>

**Comment by atilparikh** [comment]
Thanks @mkurz and team for maintaining play. Kudos.

In this release, I found that server is crashed due to heap memory stuff, once I back to 2.8.17 everything solved; but back to 2.8.18 and heap memory issues arises. Kindly look into this before releasing new one.

Thanks.

**Reply by mkurz** [reply]
Can you please open a new issue with more details how we can reproduce the problem? Thanks!

**Reply by atilparikh** [reply]
Thanks @mkurz  for quick response, as suggested https://github.com/playframework/playframework/issues/11526 created.

**Reply by mkurz** [reply]
:+1:
